src/PerspectivePanel.test.tsx

```diff
@@ -18,43 +18,43 @@ jest.mock('@grafana/runtime', () => ({
 }));
 
 class PerspectivePanel extends PerspectivePanelOriginal {
-  componentDidMount() {
-    this.viewer.current = mockedViewer;
+  async componentDidMount() {
+    (this.viewer as any).current = mockedViewer;
```

Comment:
Why not `this.viewer?.current`? It'd be consistent with the module it's testing.

Reply:
This is really a hack to get around that `current` is private

Also, you can not use `?` on the assignment side of an operation